### PR TITLE
feat(wikipedia): Theme syntax highlighting, Fix line numbers in code-blocks, and stylistic changes. (#601)

### PR DESCRIPTION
**Pull request by aspizu**
I removed the border from code-blocks as I believe it looks better without it.

### CODE REVIEW

**Review by isabelroses — APPROVED** [review]
LGTM

File: styles/wikipedia/catppuccin.user.css [hunk 1]
```diff
@@ -256,10 +256,266 @@
 
     .mw-highlight pre {
       background: @surface0 !important;
-      border-color: @surface2 !important;
+      border-width: 0;
```
Comment (isabelroses) [inline styles/wikipedia/catppuccin.user.css:259]: Oh wait sorry, just saw this. Is this avoidable?

Comment (aspizu) [reply styles/wikipedia/catppuccin.user.css:259]: Sure, can be undone if you wish.

Comment (aspizu) [reply styles/wikipedia/catppuccin.user.css:259]:
```css
border-color: @surface0 !important;
```

Comment (aspizu) [reply styles/wikipedia/catppuccin.user.css:259]: But, it makes the border look like this:

Comment (isabelroses) [reply styles/wikipedia/catppuccin.user.css:259]: Alright, that makes sense.

**Pull request merged**